test/cqlpy/run.py

```diff
@@ -271,7 +271,6 @@ def run_scylla_cmd(pid, dir):
         '--options-file',  source_path + '/conf/scylla.yaml',
         '--developer-mode', '1',
         '--ring-delay-ms', '0',
```

Comment:
> It was not invocing it anyway (--collectd 0).

It was disabling it (it's default enabled) to avoid packet spam

Reply:
Indeed, this was deliberate to avoid silly spam, although I don't remember if it was anything serious. I think we should remove this option after it no longer does anything (a Seastar change, I guess) - not while it still does something.